metadata-ingestion/src/datahub/ingestion/source/nifi.py

```diff
@@ -392,6 +400,10 @@ def __init__(self, config: NifiSourceConfig, ctx: PipelineContext) -> None:
         if self.config.ca_file is not None:
             self.session.verify = self.config.ca_file
 
+        # To keep track of process groups (containers) which have already been ingested
+        # Required, as we do not ingest all process groups but only those that have known ingress/egress processors
+        self.processed_pgs: List[str] = []
```

Comment:
this is only used when `emit_process_group_as_container` is enabled right?

Reply:
yes